starter-code/pizza.js

```diff
@@ -17,38 +14,51 @@ var state = {
   greenPeppers: true,
   whiteSauce: false,
   glutenFreeCrust: false
-}
+};
 
 // This function takes care of rendering the pizza based on the state
 // This function is triggered once at the begining and everytime the state is changed
 function renderEverything() {
-  renderPepperonni()
-  renderMushrooms()
-  renderGreenPeppers()
-  renderWhiteSauce()
-  renderGlutenFreeCrust()
-
-  renderButtons()
-  renderPrice()
+  renderPepperonni();
+  renderMushrooms();
+  renderGreenPeppers();
+  renderWhiteSauce();
+  renderGlutenFreeCrust();
+
+  renderButtons();
+  renderPrice();
 }
 
 function renderPepperonni() {
-  document.querySelectorAll('.pep').forEach(function($pep){
+  document.querySelectorAll(".pep").forEach(function($pep) {
     if (state.pepperonni) {
       $pep.style.visibility = "visible";
-    }
-    else {
+    } else {
       $pep.style.visibility = "hidden";
     }
-  })
+  });
 }
 
 function renderMushrooms() {
   // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll(".mushroom").forEach(function($mushroom) {
+    if (state.mushrooms) {
+      $mushroom.style.visibility = "visible";
+    } else {
+      $mushroom.style.visibility = "hidden";
+    }
+  });
 }
 
 function renderGreenPeppers() {
   // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll(".green-pepper").forEach(function($greenpepper) {
+    if (state.greenPeppers) {
+      $greenpepper.style.visibility = "visible";
+    } else {
+      $greenpepper.style.visibility = "hidden";
+    }
+  });
 }
 
 function renderWhiteSauce() {
```

Comment:
This can be done by adding or removing the sauce-white class from sauce section: 
  let sauce = document.querySelector(".sauce");
  if (state.whiteSauce) {
    sauce.classList.add("sauce-white");
  } else {
    sauce.classList.remove("sauce-white");
  }